### ISSUE TITLE
Basic FTP or rsync on dist directory?

### ISSUE THREAD

**Issue by ifdion** [synapticism/wordpress-gulp-starter-kit#54]
@synapticism thanks for the starter kit. The workflow is very nicely done, but IMHO it can be improved with a gulp task to push the dist directory to live server / hosting. Maybe [Vinyl FTP](https://www.npmjs.com/package/vinyl-ftp) or [Gulp rsync](https://www.npmjs.com/package/gulp-rsync) ?

**Comment by synapticism** [comment]
Yes, this would be nice to have. I'll leave this open a while and see if anyone chimes in with what solution they're using to automate deployment.
